### PR TITLE
Dev to main sprint

### PR DESCRIPTION
**Pull request by Maksim-Kotenkov**

### CODE REVIEW

File: src/VerifyQuery.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+        Returns:
+            True if the string is valid, False otherwise.
+        """
+        return (
+                VerifyQuery.validate_brackets(string) and VerifyQuery.validate_flags(string)
+                and VerifyQuery.validate_filenames(string) and VerifyQuery.validate_formats(string)
+                )
+
+# print(check("[-i=filename.png]filter[-o=out]filter"))  # should be False
```
Comment (melarozz) [inline src/VerifyQuery.py:197]: DELETE!!!@!@!@@!

File: src/VerifyQuery.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+# print(check("[-i =filename.png]filter[-o=out][-i=filename.png]filter[-o=out]"))  # should be False
+# print(check("[-i=filename.png] filter[-o=out][-i=filename.png]filter[-o=out]"))  # should be False
+#
+# print(check("[-i=filename .png]filter[-o=out][-i=filename.png]filter[-o=out]"))  # should be True
+#
+# print(check("[-i=filename.png][]][-o=out][-i=filename.png]filter[-o=out]"))  # should be False
+# print(check("[]filter[-o=out][-i=filename.png]filter[-o=out]"))  # should be False
+#
+# print(check("[-i=filename .png]filter[-o=out][out]filter[-o=out]"))  # should be True
+# print(check("[-i=filename .png]filter[-o=out ][out ]filter[-o=out]"))  # should be True but is False!!!
```
Comment (melarozz) [inline src/VerifyQuery.py:212]: DELETE ALL!!!!!!!!

File: src/filters/VideoEditor.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+from .Filter import Filter
+
+
+class VideoEditor(Filter):
+
+    def __init__(self, filter_name: str):
+        super().__init__()
+        # self.class_map: Dict[str, type] = {
+        #                                    "face_detection": FaceDetection}
+        # self.filter = self.class_map[filter_name]
```
Comment (melarozz) [inline src/filters/VideoEditor.py:16]: DELETE!!!

**Review by melarozz — CHANGES_REQUESTED** [review]

**Review by McAninKirill — APPROVED** [review]
COOOOOOOOOOOL

**Review by melarozz — APPROVED** [review]
nice

**Pull request merged**